pkg/spanconfig/spanconfigsqltranslator/sqltranslator.go

```diff
@@ -230,9 +229,18 @@ func (s *SQLTranslator) generateSpanConfigurationsForTable(
 		return nil, err
 	}
 
+	isSystemID := s.execCfg.SystemIDChecker.IsSystemID(uint32(desc.GetID()))
```

Comment:
FYI, since you already have the descriptor you can call `catalog.IsSystemDescriptor(desc)` instead.

Reply:
Done.